packages/devtools-components/src/tree.js

```diff
@@ -806,10 +806,14 @@ class Tree extends Component {
       });
     });
 
-    const style = Object.assign({}, this.props.style || {}, {
-      padding: 0,
-      margin: 0
-    });
+    const style = Object.assign(
+      {},
+      {
+        padding: 0,
```

Comment:
we should maybe check why are those needed, and maybe remove them.
It would be better to style it through CSS when/if needed so we don't have such a string specificity.

Reply:
yeah, I don't think it has any effect on the console, nor on the scopes panel.
So let's change it into: 

```js
const style = Object.assign({}, this.props.style || {});
```

and not override it in JS from src/components/PrimaryPanes/SourcesTree.js, but target the CSS class from there (i.e. `.sources-list .managed-tree .tree`).

Reply:
Thanks @nchevobbe. I'll make the suggested changes.